Lib/glob.py

```diff
@@ -124,6 +119,7 @@ def escape(pathname):
 _special_parts = ('', '.', '..')
 _dir_open_flags = os.O_RDONLY | getattr(os, 'O_DIRECTORY', 0)
 _no_recurse_symlinks = object()
+_initial_path_exists = []  # falsy sentinel
```

Comment:
```suggestion
_initial_path_exists = (,)  # falsy sentinel
```
I have not fully checked the usage yet, but an immutable sentinel might be safer

Reply:
Thanks. I've tweaked this so the boolean value doesn't matter, and so it's now `object()`